torch/fx/proxy.py

```diff
@@ -43,7 +117,14 @@ def create_node(self, kind : str, target : Target,
         if kind == 'call_function' and self.check_mutable_operations:
             check_for_mutable_operation(target, args, kwargs)
 
-        return self.graph.create_node(kind, target, args, kwargs, name, type_expr)
+        node = self.graph.create_node(kind, target, args, kwargs, name, type_expr)
+        self.node_name_to_scope[node.name] = (
+            self.scope.module_path,
+            self.scope.module_type,
+        )
```

Comment:
maybe we can add a TODO to remove this, it now becomes redundant since we have `node.meta["nn_module_stack"]`